whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -2399,6 +2399,7 @@
           "addLink": "genreForm",
           "uriTemplate": "https://id.kb.se/marc/BooksContentsType-{_}",
           "matchUriToken": "^[abcdefgijklmnopqrstuvwyz256]$",
+          "NOTE:lcMarc2bfCodeMap": "marcgt",
```

Comment:
Should these go into the typenormalization branch (are they notes on suggested mapping changes)?

Reply:
Good catch! I will remove them.